src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResource.kt

```diff
@@ -140,11 +140,11 @@ class UserResource(
   ): StaffDetail =
     userService.findByStaffId(staffId)
 
-  @PreAuthorize("hasRole('ROLE_USE_OF_FORCE')")
+  @PreAuthorize("hasRole('ROLE_USE_OF_FORCE') or hasRole('ROLE_STAFF_SEARCH')")
```

Comment:
there is a `hasAnyRole` that would probably be better

Reply:
Oh - I'll change it 'cos I can't merge it anyway!